### DISCUSSION TITLE
Should the "Manage Personal Access Tokens" be named "Applications"?

### DISCUSSION THREAD

**Discussion by erwin** [gogs/gogs#7071] [Proposal]
![image](https://user-images.githubusercontent.com/11534217/175233972-08a9537d-7681-4132-99a8-ae1f7ff6ee53.png)

On the Settings Menu, you go to "Applications" to "Manage Personal Access Tokens".
Yet only Personal Access Tokens are in there...

Another menu item is named `SSH Keys`.

Perhaps it would be best to name these `API Keys` or `Access Tokens` rather than the somewhat generic `Applications` in that menu?

I'm only trying to help. Gogs is awesome. If you dislike using the name `API Keys` or `Access Tokens` or similar in that menu, it's still an awesome app.

I just don't use Gogs every often, and every time I do use it, I literally have to search online to find where the _Personal Access Tokens_ are actually located... Are they assigned to the user? Are they assigned to the repo? I see SSH keys very clearly...

For me, ideally "API Tokens" or similar would be right below "SSH Keys" in that menu...

**Comment by unknwon** [comment]
> Perhaps it would be best to name these `API Keys` or `Access Tokens` rather than the somewhat generic `Applications` in that menu?

Good question! I think it's just so happen that only access tokens are supported as of now, but it also makes sense to rename the "Applications" to "Access tokens". PR is welcome :)
